zone/client_packet.cpp

```diff
@@ -14059,6 +14059,7 @@ void Client::Handle_OP_Taunt(const EQApplicationPacket *app)
 
 void Client::Handle_OP_TestBuff(const EQApplicationPacket *app)
 {
+	parse->EventPlayer(EVENT_TEST_BUFF, this, "", 0);
```

Comment:
We should have a rule guard here (default Handle_OP_TestBuff does nothing and returns, no quest call)

Reply:
Yeah, example.
```cpp
if (!RuleB(Character, EnableTestBuff)) {
    return;
}